eng/scripts/InstallVisualStudio.ps1

```diff
@@ -81,7 +81,7 @@ if (-not $InstallPath) {
     $vsWhere = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe"
     if (Test-Path $vsWhere)
     {
-        $installations = & $vsWhere -version '[16,17)' -format json -sort -prerelease -products $productId | ConvertFrom-Json
+        $installations = & $vsWhere -version '[16,17)' -format json -prerelease -products $productId | ConvertFrom-Json
```

Comment:
Please address the following commend from #27183:
>  we could switch to using PowerShell to sort the objects in the `$installations` array.

Without that, could end up updating a VS version the build will ignore. Issue can arise if you have multiple versions (e.g. from different channels) installed of (say) VS 2019 Enterprise.

e.g.
```suggestion
        $installations = & $vsWhere -version '[16,17)' -format json -prerelease -products $productId | ConvertFrom-Json |Sort-Object -Descending -Property installationVersion, installDate
```